testing/scalar-functions.test

```diff
@@ -39,6 +39,10 @@ do_execsql_test char {
   select char(108, 105)
 } {li}
 
+do_execsql_test char-nested {
```

Comment:
This didn't work, fixed with https://github.com/tursodatabase/limbo/pull/504/files#diff-371865d5d7b8bcaed649413c687492e61e94f21387cd9b2c47d989a033888c8bR1955